polkadot-parachains/common/src/xcm_config.rs

```diff
@@ -1,71 +1,2 @@
-use core::marker::PhantomData;
-use frame_support::{log, traits::ProcessMessageError, weights::Weight};
-use xcm::latest::prelude::*;
-use xcm_executor::traits::ShouldExecute;
-
 /// Type alias to conveniently refer to `frame_system`'s `Config::AccountId`.
 pub type AccountIdOf<R> = <R as frame_system::Config>::AccountId;
-
-//TODO: move DenyThenTry to polkadot's xcm module.
-/// Deny executing the XCM if it matches any of the Deny filter regardless of anything else.
-/// If it passes the Deny, and matches one of the Allow cases then it is let through.
```

Comment:
I finally see no reason to use this anymore. @clangenb ?

Reply:
compared it to polkadot's impl which we depend on directly now
I suspect no changes in behavior but I may have missed something. 
This release will be the basis for extended XCM testing anyway. Do you see a risk?

Reply:
Ok, I think then it is safe, probably they fixed the todo on their end. 👍